react/features/subtitles/functions.web.ts

```diff
@@ -17,3 +17,17 @@ export const notifyTranscriptionChunkReceived = (transcriptMessageID: string, la
         participant,
         ...text
     });
+
+/**
+ * Calculates the font size for the subtitles.
+ *
+ * @param {number} clientHeight - The height of the visible area of the window.
+ * @returns {number}
+ */
+export function calculateSubtitlesFontSize(clientHeight?: number) {
+    if (typeof clientHeight === 'undefined') {
+        return 16;
```

Comment:
Extract to a constant since it's used in 2 places here?

Reply:
Sure!

Reply:
fixed